modular-docs-manual/content/topics/module_guidelines-procedure.adoc

```diff
@@ -32,12 +32,14 @@ The procedure consists of one or more steps required to complete the procedure.
 
 For single-step procedures, use an unnumbered bullet instead of a numbered list.
 
+NOTE: Not all numbered lists in documentation are procedures. You can also use numbered lists in any module type for non-procedural sequences, such as a process flow of system actions.
```

Comment:
For issues #138 . 

@emmurphy1 I know we discussed putting this note in the proc template, but after I tried that, it didn't make sense to me there (strange to say something in a proc template about permitting numbered elements in other template types) and it also was cluttering the template.

Reply:
@sterobin Yes, that looks good to me.

Reply:
Thanks @bobjohnsrh